StandardsRegExt.tex

```diff
@@ -404,9 +406,10 @@ \subsection{Summary of Metadata Concepts}
        standard, it is recommended that the \xmlel{vstd:Standard} or
        \xmlel{vstd:ServiceStandard} record corresponding to the
        IVOA standard document be used to define the names.  The
-       \xmlel{vstd:StandardKeyEnumeration} might be convenient
-       for definitions that are experimental or otherwise are for
-       non-IVOA-based applications.  
+       \xmlel{vstd:StandardKeyEnumeration} was originally envisioned to
+       as a container for names.  With the adoption of Vocabularies in
+       the VO 2 \citep{2021ivoa.spec.0525D}, this type has probably
+       become obsolete.
```

Comment:
Can we be stronger here, ie obsoletize vstd:StandardKeyEnumeration in favour of a controled vocabulary ?